File: 1117.py
cont = 0
soma = 0
while cont != 2:
    n = float(input()) 
    if 0.0 <= n <= 10.0:
       soma += n
       cont += 1
    else:
        print("nota invalida")
    
print("media = {:.2f}".format(soma/float(cont)))   

File: solutions/1117.cpp
#include<iostream>
#include <stdio.h>
using namespace std;
int main(){
    float n, m=0, c=0;
    bool cond = true;
    while(cond){
        cin >> n;
        if(n<0 || n>10) cout << "nota invalida\n";
        else{
            m+=n;
            c++;
        }
        if(c==2) cond = false;
    }
    printf("media = %.2f \n", m/c);
    return 0;
}


